File: Components/Reporting/CS/DevExpress.DemoReports/Data/SalesPerson.cs
using System;
using System.Collections.Generic;

namespace DevExpress.DemoData.Models
{
    public partial class SalesPerson
    {
        public long OrderID { get; set; }
        public string Country { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string ProductName { get; set; }
        public string CategoryName { get; set; }
        public Nullable<System.DateTime> OrderDate { get; set; }
        public decimal UnitPrice { get; set; }
        public short Quantity { get; set; }
        public double Discount { get; set; }
        public decimal ExtendedPrice { get; set; }
        public string FullName { get; set; }
    }
}

File: Components/Reporting/VB/DevExpress.DemoReports/Data/SalesPerson.vb
Imports System
Imports System.Collections.Generic

Namespace DevExpress.DemoData.Models
    Partial Public Class SalesPerson
        Public Property OrderID() As Long
        Public Property Country() As String
        Public Property FirstName() As String
        Public Property LastName() As String
        Public Property ProductName() As String
        Public Property CategoryName() As String
        Public Property OrderDate() As Date?
        Public Property UnitPrice() As Decimal
        Public Property Quantity() As Short
        Public Property Discount() As Double
        Public Property ExtendedPrice() As Decimal
        Public Property FullName() As String
    End Class
End Namespace
